### DISCUSSION TITLE
Viewer is watching from the future?

### DISCUSSION THREAD

**Discussion by TinFoilFox** [owncast/owncast#1882] [Q&A] [answered]
I host a small movie night for my friends every Friday night. Recently, whenever a certain viewer joins the stream, his browser reports negative latency. Is this something on my end? I am hosting Owncast on the free ARM64 server Oracle offers along with their free object storage and so I have to pull and build the docker image to get it to run. This hasn't been an issue until recent streams or I might just not have noticed it until now.

Is this a bug or have I configured something wrong?

![Stream_Latency](https://user-images.githubusercontent.com/15332259/165172286-0f3a7bef-90e3-4c9b-9b40-ef36c66741c3.png)

**Comment by gabek** [comment]
Thanks for checking in. This feature hasn't been released yet, but it's good to hear how it's working for you in testing. Do you know who this person is? I'm curious if their clock is set incorrectly.

**Reply by TinFoilFox** [reply]
I do not know the viewer personally, however he is a friend of a friend and I could chat with him the next time I stream.

Interesting edge case I guess. Would this just be caused by their system clock being 2 minutes ahead? I guess that makes the most sense. Is their anyway to sanity check latency? Either way, thanks for the response, and I'll see if he has disabled the auto-set Time & Date in windows or something.

**Comment by gabek** [comment, marked as answer]
Yeah, I plan on just throwing out values that seem excessive. While that's not a great solution, it's better than letting bad values throw things off.
